### PR TITLE
Adds 7vs7 robot usage and misusage

### PR DESCRIPTION
**Pull request by PatrickGoettsch**

### CODE REVIEW

File: content/technical_challenges.tex [hunk 1]
```diff
@@ -39,6 +39,30 @@ \subsection{7 vs. 7}
...
 
+            \begin{hintbox*}{Robot usage}
+                All teams using a robot from the robot pool have to take care of the robot and try to prevent any damage from the robot. A safe operation of the robots is paramount. Referees should act to prevent damage to robots, and be proactive in enforcing rules designed to minimize damage. Teams should endeavor to install bug-free and conservative software. This challenge features elements of speed and competitive game play, there are strict consequences for damaging robots, and we hope no team will be prevented from competing due to these reasons.
+                All teams an encouraged to implement features in their code to prevent falling or when falling to fall less destructive. We ask all teams to exchange ideas on that in our \textit{RC-SPL Discord}.
```
Comment (arnemoos) [inline content/technical_challenges.tex:44]: All teams **are** encouraged

File: content/technical_challenges.tex [hunk 2]
```diff
@@ -39,6 +39,30 @@ \subsection{7 vs. 7}
...
+            \end{hintbox*}
+        
+        \paragraph{Robot pool disqualification}
+            All teams using robots for the robot pool are encouraged to implement robot protection features to prevent abnormal damages and abnormal probability of a fall robot. The teams providing robots trust in the teams using their robots that they operate them with care.
```
Comment (arnemoos) [inline content/technical_challenges.tex:48]: of a fall**en** robot.

**Review by arnemoos — CHANGES_REQUESTED** [review]

File: content/technical_challenges.tex (hunk 1 above)
Comment (PatrickGoettsch) [reply content/technical_challenges.tex:44]: Done

File: content/technical_challenges.tex (hunk 2 above)
Comment (PatrickGoettsch) [reply content/technical_challenges.tex]: Done

**Pull request merged**